### PR TITLE
Add uint32 and uint64 to ? output

### PR DESCRIPTION
**Pull request by afjoseph**
Closes https://github.com/radare/radare2/issues/13883

### CODE REVIEW

File: libr/core/cmd_help.c [hunk 1]
```diff
@@ -553,8 +553,10 @@ static int cmd_help(void *data, const char *input) {
 				} else {
 					if (n >> 32) {
 						r_cons_printf ("int64   %"PFMT64d"\n", (st64)n);
+						r_cons_printf ("uint64  %"PFMT64u"\n", (ut64)n);
 					} else {
 						r_cons_printf ("int32   %d\n", (st32)n);
+						r_cons_printf ("uint32  %d\n", (ut32)n);
```
Comment (radare) [inline libr/core/cmd_help.c]: %d is signed, better use %u

**Comment by radare** [comment]
also update the tests and fix windows issue

**Comment by radare** [comment]
closing, as  long as there is another pr submmited before and adding the same fields for json

**Comment by afjoseph** [comment]
Apologies. Didn't see that PR open before

**Comment by radare** [comment]
Np. There are zillions of issues opened. Its funny that u too tried to solve it at the same time :D

**Pull request closed without merging**